count-stats/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
There is a `.gitignore` at the project level that is applied to the modules. Please remove this file and use the more general one

Reply:
removed